operations/host_spawn.go

```diff
@@ -563,23 +563,43 @@ func hostStart() cli.Command {
 func hostSSH() cli.Command {
 	const (
 		identityFlagName = "identity_file"
+		dryRunFlagName   = "dry-run"
 	)
 
 	return cli.Command{
 		Name:  "ssh",
 		Usage: "ssh into a spawn host",
+		UsageText: `
+SSH into a host by its ID and optionally pass arbitrary additional parameters to the SSH binary.
+
+Examples:
+* SSH into a host (i-abcdef12345) by ID:
+	evergreen host ssh --host i-abcdef12345
+
+* As a special case, if you pass a single argument, it will be interpreted as the host ID:
+	evergreen host ssh i-abcdef12345
+
+* SSH into a host (i-abcdef12345) and pass additional arguments to the SSH binary to use verbose mode (-vvv) and run a command (echo hello world):
+	evergreen host ssh --host i-abcdef12345 -- -vvv "echo hello world"
+`,
+		ArgsUsage: "[-- other_ssh_arguments]",
```

Comment:
I don't see this in the output of `evergreen ssh -h` ; do you know where/if this gets used?

Reply:
Hmm I think it might not do anything. I swear it did something when I originally added it, but it's probably not necessary since the usage text includes examples of how to use the command, so I removed this.